browser.js

```diff
@@ -78,9 +77,8 @@ function pbkdf2Sync (password, salt, iterations, keylen, digest) {
       }
     }
 
-    var destPos = (i - 1) * hLen
-    var len = (i === l ? r : hLen)
-    T.copy(DK, destPos, 0, len)
```

Comment:
`var len` is probably important, but, none of our tests cover it actually doing anything useful.
So uh... ?